wordpress-mobile/WordPress-iOS
# Blogging Reminders - Time Selection UI

Fixes #NA

To test:
Test on iPhone and iPad
- Build/run and go to Site Settings -> Blogging Reminders
- Tap on at least one day in the settings screen and make sure that the time selection button shows up (see screenshot below)
- Tap on the button and make sure that the time selection view shows up (see screenshots below for iPhone and iPad)
- tap on Back and make sure you return correctly to the settings screen

**NOTE: this PR does not actually set the Blogging Reminders time, at this time. It only contains the UI**

Time Selection Button | Time Selection View - iPhone | Time Selection View - iPad
-- | -- | --
<img width="300"  src="https://user-images.githubusercontent.com/34376330/129298833-0be7a95c-5231-48e1-a83f-7d3e8f77b4f0.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/129298837-ca537114-18c9-44b5-ae25-9dfa2df23d32.png"> | <img width="300" src="https://user-images.githubusercontent.com/34376330/129298847-f33f10a1-cf69-40fe-9c9f-1f19a1175ef3.png">

## Regression Notes
1. Potential unintended areas of impact
This adds new types and a new element in Blogging Reminders view. The only potential regression could be UI bugs in blogging reminders

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test Blogging Reminders UI on both iPhone and iPad

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
